docs/source/markdown/podman-run.1.md.in

```diff
@@ -692,8 +691,8 @@ $ podman run --security-opt unmask=/foo/bar:/sys/firmware fedora bash
 
 ### Setting device weight
 
-If you want to set _/dev/sda_ device weight to **200**, you can specify the device
-weight by **--blkio-weight-device** flag. Use the following command:
+Specify the device weight by **--blkio-weight-device** flag.
+Use the following command:
```

Comment:
The original "sda" and "200" text is lost, so instead of "Use the following command", it might be better to just say "Example:"